sharkfuser/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ include(CTest)
 list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_LIST_DIR}/build_tools/cmake/
 )
-include(CTestMacros)
-include(SharkFuserUtils)
+include(FusilliBuildMacros)
+include(FusilliTestTargets)
```

Comment:
IMHO `FusilliTestTargets` is a little confusing, the file doesn't define any targets. I'd name these  `FusilliBuildUtils` an `FusilliTestUtils` or just `BuildUtils` and `TestUtils`